app/views/layouts/_navigation.html.erb

```diff
@@ -4,7 +4,7 @@
     <% if user_signed_in? %>
     <%= link_to "<span>Dashboard</span>".html_safe, app_url, :class=>"link-home" %>
```

Comment:
Looks like you can add `, "aria-label" => "Home-Logo"` to this line too.

The relevant CSS shows they use the same logo as the background image:
```
&.link-home {
      float: left;
      margin-left: 5px;
      width: 130px;
      background: url(/assets/mylibrarynyc-logo-white-mini.png) center no-repeat;
```

Reply:
Adding it.